### ISSUE TITLE
Moving namespace from no project to a project fails silently

### ISSUE THREAD

**Issue by pib** [rancher/terraform-provider-rancher2-archive#78]
I have a namespace that is currently not in a project (the namespace was pre-existing in an imported cluster), and I used `terraform import` to import a `rancher2_namespace` that I had defined in terraform.

I ran `terraform apply` and it showed that it was going to change like:
```
Terraform will perform the following actions:

  ~ rancher2_namespace.ns
      project_id: "c-abcde" => "c-abcde:p-vwxyz"
```
Then after the apply finished, the namespace is still not in the project, but the rancher2 provider is acting like it is (running `terraform apply` again results in it saying there are no changes to be made).

I manually moved the namespace to another project and then ran apply again and it moved it to the desired project, so it seems to only be an issue when the namespace isn't currently in a project.

**Comment by rawmind0** [comment]
Fix on PR #85

**Comment by rawmind0** [comment]
Please reopen if needed.
